modules/opentracing-cpp/1.6.0/presubmit.yml

```diff
@@ -0,0 +1,34 @@
+verify_linux_targets:
```

Comment:
The syntax is wrong, all jobs must be under `tasks:`,

Check https://github.com/bazelbuild/continuous-integration/tree/master/buildkite#basic-syntax